packages/attestation-service/src/db.ts

```diff
@@ -219,3 +225,49 @@ async function purgeExpiredRecords() {
     rootLogger.error({ err }, 'Cannot purge expired records')
   }
 }
+
+// Do the health check to update the gauge
+async function tryHealthCheck() {
+  try {
+    const failureReason = await doHealthCheck()
+    if (failureReason) {
+      rootLogger.warn(`Health check failed: ${failureReason}`)
+    }
+  } catch {
+    rootLogger.warn(`Health check failed`)
+  }
+}
+
+// Check health and return failure reason, null on success.
+export async function doHealthCheck(): Promise<string | null> {
```

Comment:
Should this live in this `db` file?

Reply:
I considered moving it, but `db` has always had everything to do with ck and the full node in it too. will consider splitting in future